### DISCUSSION TITLE
LLMs Use Cases

### DISCUSSION THREAD

**Discussion by sethjuarez** [community/community#59354] [Programming Help]
Hello friends! I'm curious about your experiences with LLMs and how you've utilized them. Are there any exciting projects that you've worked on that we should all check out?

**Comment by DeanCoutinho** [comment]
For medical use cases, I encourage you to check out John Snow Lab's MEDICAL LLM blogs: https://www.johnsnowlabs.com/large-language-models-blog/

**Comment by rohankshah04** [comment]
Hey folks - I'm a sophomore at UPenn studying Math/CS. My friends and I just launched our open source python package discus. The package helps developers create high-quality synthetic data to fine-tune/train LLMs. I think there is a ton of potential use cases for this. And we'll continue to makes new updates.
If you're fine-tuning models and struggling with data, discus is for you. Please give it a try and let us know if you have any feedback. Also, would really appreciate if you all could send it around to AI builders/developers. Try it out [here](https://github.com/discus-labs/discus-synthetics):

**Comment by morpheuslord** [comment]
LLMs have great use cases in cybersecurity. It has so much in store that I wrote 4 conference papers and 2 research papers on it. In all seriousness, LLM models have the capacity to write and also when worked with can be used to write commands and execute commands making automation a breeze also using it as a day-to-day assistant for log and scan analysis for crucial details makes work easy.
